README.md

```diff
@@ -0,0 +1,14 @@
+# Sudoku Board Finder
+
+Given a screenshot of a sudoku board game like on [/r/sudoku](https://www.reddit.com/r/sudoku/), 
+find the board and numbers on the board and return a link to a web sudoku solver like [sudokuwiki.org](sudokuwiki.org).
+
+Example input:
+
+<img src="https://i.redd.it/gqg09x5yaxl11.png" width=300>
+
+Example output:
+
+```
+http://www.sudokuwiki.org/sudoku.htm?bd=150763009926005000307920561513070000600831057870500003200300045400050802705200006
```

Comment:
The link doesn't match the image. There is an additional 2 in D6 which does not appear in the image. Please fix.

Reply:
Fixed with 406263324a274b777434fbe542b7a7282dd14d9c